saleor/plugins/tests/sample_plugins.py

```diff
@@ -255,7 +255,9 @@ def promotion_created(self, promotion: "Promotion", previous_value: Any):
     def promotion_updated(self, promotion: "Promotion", previous_value: Any):
         return None
 
-    def promotion_deleted(self, promotion: "Promotion", previous_value: Any):
+    def promotion_deleted(
+        self, promotion: "Promotion", previous_value: Any, webhooks=None
```

Comment:
These and other changes with the `webhooks` param are not related to `call_event`, are they? Looks like separate PR?

Reply:
moved to separate pr